File: carpartsshop/angularclient/src/app/model/car/car.ts
import {BaseEntity} from "../BaseEntity";

export class Car extends BaseEntity{
  make: string | undefined;
  model: string | undefined;
  fuelType: string | undefined;
  year: BigInteger | undefined;


  constructor(id: number, make: string | undefined, model: string | undefined, fuelType: string | undefined, year: BigInteger | undefined) {
    super(id);
    this.make = make;
    this.model = model;
    this.fuelType = fuelType;
    this.year = year;
  }
}

File: carpartsshop/src/main/java/fsega/project/carpartsshop/model/Car.java
package fsega.project.carpartsshop.model;

import jakarta.persistence.Entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Entity
public class Car extends BaseEntity{
    private String make;
    private String model;
    private String fuelType;
    private Integer year;
}
